package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "grunt next && next",
     "build": "grunt next && next build && next export",
     "start": "next start",
+    "format": "prettier --write \"{components,pages,utils}/**/*.js\"",
```

Comment:
Yay! You could also use single quotes for this, if you like (they're fully supported in bash etc.)